com.nolimitconnect.NoLimitConnect.yml

```diff
@@ -0,0 +1,41 @@
+
+app-id: com.nolimitconnect.NoLimitConnect
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: nolimitconnect
+finish-args: 
+    - --device=all
+    - --share=ipc
+    - --share=network
+    - --socket=fallback-x11       
+    - --socket=wayland
+    - --socket=pulseaudio
+    - --filesystem=host
```

Comment:
host is very broad. Why do you need it?

Reply:
Partially this is from https://github.com/flathub/org.qbittorrent.qBittorrent/blob/master/org.qbittorrent.qBittorrent.yaml
but I am not absolutely certain I need all of host. What I need is the ability to read all media types like videos, pictures music etc from anywhere the user has access to for user to add to library and/or share
I do not need write access anywhere except in the normal sandbox for downloading and media caching etc.
What do you think? can I get read access anywhere in home without the filesystem=host

Reply:
Does it support filechooser portals ? https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.FileChooser.html

Reply:
It does not support filechooser portals. It uses mostly c style fopen fread fclose style file io and some use of Qt QFile. It compiles on Windows/Linux/Android which means as generic of code as possible mostly.